event_routing_backends/processors/xapi/event_transformers/problem_interaction_events.py

```diff
@@ -152,6 +152,27 @@ def get_result(self):
             }
         )
 
+    def get_object(self):
+        """
+        Get object for xAPI transformed event.
+
+        Returns:
+            `Activity`
+        """
+        language = self.find_nested(self.event, 'accept_language', default=constants.EN).split(',')[0]
```

Comment:
The found string would always be separated by comma.?

Reply:
no, but if there is no comma with return the whole string